src/components/tests/Home.test.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import Home from '../Home';
+
+const home = (
+  <Home />
+);
+
+it('renders correctly', () => {
+  expect(home).toMatchSnapshot();
+});
```

Comment:
- [ ] It might be a good idea to fix the tests because they are failing.

![test-failing](https://user-images.githubusercontent.com/79876959/147312183-e2106825-682a-4158-8cb6-bb8485a8b066.PNG)

Reply:
✔️